Makefile

```diff
@@ -259,6 +259,10 @@ smoketest:
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=stm32f4disco        examples/blinky2
 	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=stm32f4disco-1       examples/blinky1
+	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=stm32f4disco-1       examples/blinky2
+	@$(MD5SUM) test.hex
```

Comment:
Only one smoke test should be fine. The only thing that an extra smoke test tests is that the `LEDx` constants exist, which is not something that is likely to break unintentionally (and it slows down tests).

Reply:
Removed extra smoketest.